src/components/paradox/results/personality/career/container/style.scss

```diff
@@ -0,0 +1,13 @@
+@import "~style/helpers";
+
+.container {
+  @extend %widgetContainer;
+  margin: 0 auto 20px;
```

Comment:
Design told my Paradox prefers to use multiples of 8 for spacing, so using various `$buffer` variables or using a multiplier with $buffer would be good:

https://github.com/traitify/traitify-widgets/blob/attract/src/style/paradox/variables.scss#L20

Reply:
I'm not sure. I just remember Dave emphasizing it when I was working with him in the office

Reply:
Ok adjusted! Now using $buffer for most of the margins and padding!